src/integrations/admin/first-time-configuration-integration.php

```diff
@@ -309,6 +339,22 @@ private function get_company_logo_id() {
 		return $this->options_helper->get( 'company_logo_id', '' );
 	}
 
+	/**
+	 * Gets the company logo id from the option in the database.
+	 *
+	 * @param int|string $company_logo_id The given company logo id by the user, default empty.
+	 *
+	 * @return string The company logo id.
+	 */
+	private function get_company_fallback_logo( $company_logo_id ) {
+		if ( $company_logo_id ) {
+			return false;
+		}
+		$logo_id = $this->meta_tags_context->fallback_to_site_logo();
```

Comment:
I think a URL will be passed to this function and a ( 'boolean' | 'string' ) will be returned.
```suggestion
	 * Gets the company logo url from the option in the database.
	 *
	 * @param string $company_logo The given company logo by the user, default empty.
	 *
	 * @return {boolean | string} The company logo URL.
	 */
	private function get_company_fallback_logo( $company_logo ) {
		if ( $company_logo ) {
			return false;
		}
		$logo_id = $this->meta_tags_context->fallback_to_site_logo();
```